.github/workflows/docker-publish.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         uses: docker/login-action@v3
         with:
           registry: ${{ env.REGISTRY }}
-          username: ${{ secrets.QUAY_USERNAME }}
+          username: ${{ vars.QUAY_USERNAME }}
```

Comment:
where is this variable getting added?

Reply:
Hi @mkumatag,
We may need to add this variable in repository settings to have the same defined.